Jenkinsfile.security

```diff
@@ -0,0 +1,6 @@
+#!groovy
```

Comment:
how does this file get called? not for every PR, right?

Reply:
You can check the actual pipeline in `cdis-jenkins-lib`.

It will run every Saturday: https://github.com/uc-cdis/cdis-jenkins-lib/blob/master/vars/securityPipeline.groovy#L14-L15

And it will skip scan every two Saturdays: https://github.com/uc-cdis/cdis-jenkins-lib/blob/master/vars/securityPipeline.groovy#L21-L42